src/test/scala/uk/gov/nationalarchives/tdr/api/model/consignment/CreateConsignmentReferenceSpec.scala

```diff
@@ -0,0 +1,22 @@
+package uk.gov.nationalarchives.tdr.api.model.consignment
+
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import scala.concurrent.ExecutionContext
+
+class CreateConsignmentReferenceSpec extends AnyFlatSpec with ScalaFutures with Matchers {
+  implicit val executionContext: ExecutionContext = ExecutionContext.Implicits.global
+
+  "createConsignmentReference" should "create a full consignment reference when given a numerical ID" in {
+    val consignmentReferenceClass = new CreateConsignmentReference
+    val createdYear = 2020
+    val consignmentSequence = 5000
+
+    val result: String = consignmentReferenceClass.createConsignmentReference(createdYear, consignmentSequence)
+
+    result should include ("LBB")
+    result should equal ("TDR-2020-LBB")
```

Comment:
It would be really great to have tests for different edge cases here. Give me a shout and I'll show you a ScalaTest trick for testing multiple examples without writing lots of repetitive tests.

Reply:
That would be great yeah, thanks! I'll shoot you a message :smile: